interface/ceed-operator.c

```diff
@@ -1067,7 +1067,6 @@ int CeedOperatorLinearAssembleAddDiagonal(CeedOperator op, CeedVector assembled,
       ierr = CeedOperatorCreateFallback(op); CeedChk(ierr);
     }
     // Assemble
-    ierr = CeedVectorSetValue(assembled, 0.0); CeedChk(ierr);
```

Comment:
Thanks for catching this. I'm hoping we can get this branch and these various small Bugfixes into master soon